docs/kusama-social-recovery.md

```diff
@@ -0,0 +1,141 @@
+---
+id: kusama-social-recovery
+title: Social Recovery
+sidebar_label: Account Recovery
+---
+
+Managing an account is not an easy task. Many people have lost their private key due to improper key
+management over the past few years. Kusama provides a method that allows users to recover their
+accounts by setting up a social recovery. It is an M-of-N recovery tool that is based on the
+multisignature wallet to get back access of your lost account.
+
+> Note: There is no way to get back your private key by using this method. This is just a way of
+> performing transactions on behalf of the lost account, so you can think of it like a proxy
+> instead.
+
+In this guide, you will learn how to create a recoverable account, how to recover it, and what you
+need to be aware of when using it.
+
+## Create a Recoverable Account
+
+You will use your existing account to call `createRecovery` to select a list of accounts that you
+trust to help you recover the account when you lost the private key. To create a recoverable
+account, you will be required to set a `threshold` that is the number of your friends who need to
+approve the recovery process in order to recover your account.
+
+![](/img/recovery/social-recovery-diag-1.png)
+
+First, go to [PolkadotJS->Accounts](https://polkadot.js.org/apps/#/accounts) page that shows all
+available accounts on your browser's local storage. To create a recoverable account, making sure
+that you have some KSMs to pay for the transaction fees and some will be used for reserved as well.
+
+![](/img/recovery/social-recovery-1.png)
+
+Then click the menu that is located besides the `send` button, and choose `Make recoverable`.
+
+![](/img/recovery/social-recovery-2.png)
+
+Now you need to provide the following information:
+
+`trusted social recovery helpers` - A list of accounts that you trust. These can help you if you
+lose the private key. Since setting up a recoverable account is required to lock up KSMs, better to
+ensure your account has enough transferable balance to cover it. As you select additional recovery
+helpers, more KSMs will be required to stay locked up.
```

Comment:
```suggestion
`trusted social recovery helpers` - A list of accounts that you trust. These can help you if you
lose the private key. Since setting up a recoverable account requires you to lock up KSM, ensure 
your account has enough transferable balance to cover it. As you select additional recovery
helpers, more KSM will be required.
```